pkgs/stdenv/darwin/default.nix

```diff
@@ -297,7 +297,7 @@ in rec {
     initialPath = import ../common-path.nix { inherit pkgs; };
     shell       = "${pkgs.bash}/bin/bash";
 
-    cc = import ../../build-support/cc-wrapper {
+    cc = lib.callPackageWith {} ../../build-support/cc-wrapper {
```

Comment:
It adds the override field so I can replace cc while leaving everything else the same.